### PR TITLE
Makes defibrillators visible on cryo pods when applied.

### PR DESCRIPTION
**Pull request by 112358sam**
## About the PR 
This PR makes it so the defibrillator is visible on the cryo pod when added. The defibrillator animates like the handheld version when used on the interior occupant. 

https://user-images.githubusercontent.com/22460970/155864086-ff198294-5380-446f-b21a-e837af25736e.mp4

Two small related bugs were also fixed. One is that defibrillators were being ejected with the occupant when using the right click 'move eject' button. The proper way to remove the defibrillator is with a wrench. The other is that there was flavor text describing placing the electrodes on the person's chest even when the defibrillator was used in a cryo pod. Now it checks and skips that description if the defibrillator is part of the cryo pod.

## Why's this needed? 
A clear visual indicator that the cryo pod has a defibrillator will both help make it more clear that someone has already set it up and provide an indicator to new players that cryo pods can accept defibrillators.

## Changelog

```changelog
(u)112358sam
(+)Cryo pods now show the defibrillator on their sprite if they have one.
```

### CODE REVIEW

**Comment by 112358sam** [comment]
Assuming these errors are issues with the runtime checker. No problems on the local branch. Will rerun in a few hours.

File: code/modules/atmospherics/machinery/unary/cryo_cell.dm [hunk 1]
```diff
@@ -298,7 +301,19 @@
 
 
 	proc/add_overlays()
-		src.overlays = list(O1)
+		src.overlays = list(O1, O2)
```
Comment (pali6) [inline code/modules/atmospherics/machinery/unary/cryo_cell.dm:304]: no, use UpdateOverlays

**Review by pali6 — CHANGES_REQUESTED** [review]

File: code/modules/atmospherics/machinery/unary/cryo_cell.dm (hunk 1 above)
Comment (112358sam) [reply code/modules/atmospherics/machinery/unary/cryo_cell.dm]: Done!

File: code/modules/atmospherics/machinery/unary/cryo_cell.dm [hunk 2]
```diff
@@ -10,7 +10,8 @@
 	var/on = 0
 	var/datum/light/light
 	var/ARCHIVED(temperature)
-	var/obj/overlay/O1 = null
+	var/image/bottom_image = null
```
Comment (pali6) [inline code/modules/atmospherics/machinery/unary/cryo_cell.dm:13]: These could be removed and replaced by SafeGetOverlayImage calls

**Review by pali6 — APPROVED** [review]

File: code/modules/atmospherics/machinery/unary/cryo_cell.dm (hunk 2 above)
Comment (112358sam) [reply code/modules/atmospherics/machinery/unary/cryo_cell.dm]: I swapped over to SafeGetOverlayImage, but I needed to add some additional features (pixel_x and pixel_y) to the function.

**Review by pali6 — APPROVED** [review]

**Pull request merged**